backend/app/routes/folders.py

```
@@ -417,14 +417,23 @@ def disable_ai_tagging(request: UpdateAITaggingRequest):
     response_model=DeleteFoldersResponse,
     responses={code: {"model": ErrorResponse} for code in [400, 500]},
 )
-def delete_folders(request: DeleteFoldersRequest):
+def delete_folders(
+    request: DeleteFoldersRequest, app_state: State = Depends(get_state)
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)